examples/3.tutorials/lit-05-SimpleMotion.jl

```diff
@@ -0,0 +1,123 @@
+obj.Δw .= 0 # hide
+
+# The `SimpleMotion` model includes a list of `SimpleMotionType`'s, to enabling mix-and-matching simple motions.
+# These are [`Translation`](@ref), [`Rotation`](@ref), [`HeartBeat`](@ref) and their periodic versions
+# [`PeriodicTranslation`](@ref), [`PeriodicRotation`](@ref) and [`PeriodicHeartBeat`](@ref).
+
+# ### Head Translation
+#
+# In this example, we will add a [`Translation`](@ref) of 2 cm in x, with duration of 200 ms (v = 0.1 m/s):
+
+obj.motion = SimpleMotion([
+    Translation(t_start=0.0, t_end=200e-3, dx=2e-2, dy=0.0, dz=0.0)
+])
+p1 = plot_phantom_map(obj, :T2 ; height=450, intermediate_time_samples=4) # hide
+
+#md savefig(p1, "../assets/5-phantom1.html") # hide
+#jl display(p1)
+
+#md # ```@raw html
+#md # <center><object type="text/html" data="../../assets/5-phantom1.html" style="width:85%; height:470px;"></object></center>
+#md # ```
+
+## Read Sequence # hide
+seq_file1 = joinpath(dirname(pathof(KomaMRI)), "../examples/5.koma_paper/comparison_accuracy/sequences/EPI/epi_100x100_TE100_FOV230.seq") # hide
+seq1 = read_seq(seq_file1) # hide
+
+## Simulate # hide
+raw1 = simulate(obj, seq1, sys) # hide
+
+## Recon # hide
+acq1 = AcquisitionData(raw1) # hide
+acq1.traj[1].circular = false # hide
+Nx, Ny = raw1.params["reconSize"][1:2] # hide
+reconParams = Dict{Symbol,Any}(:reco=>"direct", :reconSize=>(Nx, Ny)) # hide
+image1 = reconstruction(acq1, reconParams) # hide
+
+# If we simulate an EPI sequence with acquisition duration (183.989 ms) comparable with the motion's duration (200 ms),
+# we will observe motion-induced artifacts in the reconstructed image.
+## Plotting the recon # hide
+p2 = plot_image(abs.(image1[:, :, 1]); height=400) # hide
+#md savefig(p2, "../assets/5-recon1.html") # hide
+#jl display(p2)
+
+#md # ```@raw html
+#md # <center>
+#md # <object type="text/html" data="../../assets/5-recon1.html" style="width:65%; height:420px;"></object>
+#md # </center>
+#md # ```
+
+# The severity of the artifacts can vary depending on the acquisition duration and $k$-space trajectory.
+
+# ### Motion-Corrected Reconstruction
+# 
+# Once simulation is done, it is possible to perform a corrected reconstrution 
+# in order to revert the motion effect in the final image. 
+# This can be achieved by multiplying each sample of the acquired signal 
+# by a phase which is proportional to the displacement in each direction (Δx, Δy, Δz)
+# at the time instant when the sample was acquired [[Godenschweger, 2016]](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4930872/):
+
+# ```math
+# S(k_x, k_y, k_z)_{\text{cor}} = S(k_x, k_y, k_z)_{\text{orig}} \cdot e^{i \Delta \phi_{\text{cor}}} = S(k_x, k_y, k_z)_{\text{orig}} \cdot e^{i 2 \pi (k_x \Delta x + k_y \Delta y + k_z \Delta z)}
+# ```
+
+# We need to obtain the displacements in every ADC sampling time of the sequence.
+# Since translation is a rigid motion, 
+# we can obtain the displacements only for one spin, 
+# as the displacements of the rest will be the same.
```

Comment:
```suggestion
# In practice, we would need to estimate or measure the motion before performing a motion-corrected reconstruction, but for this example, we will directly use the displacement functions ``\boldsymbol{u}(\boldsymbol{x}, t)`` defined by `obj.motion::SimpleMotion`. 
# Since translations are rigid motions (``\boldsymbol{u}(\boldsymbol{x}, t)=\boldsymbol{u}(t)`` no position dependence), we can obtain the required displacements by calculating ``\boldsymbol{u}(\boldsymbol{x}=\boldsymbol{0},\ t=t_{\mathrm{adc}})``.
```